src/hooks/useEditVideoSheet.tsx

```diff
@@ -30,6 +30,8 @@ type ContextValue = {
   setSelectedVideoTabCachedAssets: (files: InputFilesState) => void
   sheetState: EditVideoSheetState
   setSheetState: (state: EditVideoSheetState) => void
+  haveVideoTabsAssetCache: boolean
+  hasVideoTabAssetCache: (tabIdx: number) => boolean
```

Comment:
I think we need to improve the naming. It's hard to tell which is a boolean and which a function. How about `anyVideoTabsCachedAssets' for the boolean and 'hasVideoTabAnyCachedAssets'? A bit explicit but should be more readable